src/services/refactors/generateGetAccessorAndSetAccessor.ts

```diff
@@ -0,0 +1,165 @@
+        const accessorModifiers = hasModifiers ? (
+            (getModifierFlags(propertyDeclaration) & ModifierFlags.Private || !propertyDeclaration.modifiers) ? createNodeArray([createToken(SyntaxKind.PublicKeyword)]) : propertyDeclaration.modifiers
+        ) : undefined;
+
+        const getAccessor = generateGetAccessor(fieldName, accessorName, accessorType, accessorModifiers);
+        const setAccessor = generateSetAccessor(fieldName, accessorName, accessorType, accessorModifiers);
+
+        const modifiers = hasModifiers ? createNodeArray([createToken(SyntaxKind.PrivateKeyword)]) : undefined;
+        if (needUpdateName || needUpdateModifiers) {
+            changeTracker.replaceNode(file, propertyDeclaration, updateoriginalPropertyDeclaration(propertyDeclaration, fieldName, modifiers), {
+                suffix: newLineCharacter
+            });
+        }
+
+        changeTracker.insertNodeAfter(file, propertyDeclaration, getAccessor);
+        changeTracker.insertNodeAfter(file, propertyDeclaration, setAccessor);
+
+        return {
+            edits: changeTracker.getChanges(),
+            renameFilename: undefined,
+            renameLocation: undefined,
+        };
+    }
+
+    function getConvertibleFieldAtPosition(file: SourceFile, startPosition: number): Info | undefined {
+        const node = getTokenAtPosition(file, startPosition, /*includeJsDocComment*/ false);
+        const propertyDeclaration = findAncestor(node.parent, isPropertyDeclaration);
+
+        if (!propertyDeclaration || propertyDeclaration.name.kind !== SyntaxKind.Identifier) return undefined;
+        // make sure propertyDeclaration have only AccessibilityModifier
+        if ((getModifierFlags(propertyDeclaration) | ModifierFlags.AccessibilityModifier) !== ModifierFlags.AccessibilityModifier) return undefined;
+
+        const containerClass = getContainingClass(propertyDeclaration);
+        if (!containerClass) return undefined;
+
+        const members = getMembersOfDeclaration(containerClass);
+        if (!members) return undefined;
+
+        const needUpdateName = propertyDeclaration.name.text.charCodeAt(0) !== CharacterCodes._;
+
+        const accessorName = needUpdateName ? propertyDeclaration.name.text : propertyDeclaration.name.text.substring(1);
+        const fieldName = `_${accessorName}`;
+
+        if (find(members, member => needUpdateName ? member.name.getText() === fieldName : member.name.getText() === accessorName)) return undefined;
+
+        const hasModifiers = !!find(members, member => !!member.modifiers);
+        const needUpdateModifiers = hasModifiers && (!propertyDeclaration.modifiers || !hasModifier(propertyDeclaration, ModifierFlags.Private));
+        const accessorType = propertyDeclaration.questionToken ? mergeTypeNodeToUnion(propertyDeclaration.type, createKeywordTypeNode(SyntaxKind.UndefinedKeyword)) : propertyDeclaration.type;
+
+        return {
+            originalName: propertyDeclaration.name.text,

```

Comment:
Might want to make `const originalName` early since it's used in 3 other places.